docs/sources/notify/slack/index.md

```diff
@@ -40,6 +40,55 @@ For Open Source Grafana OnCall Slack installation guidance, refer to
 1. Provide your Slack workspace URL and sign with your Slack credentials.
 1. Click **Allow** to give Grafana OnCall permission to access your Slack workspace.
 
+## Why does OnCall Slack App require so many permissions?
+OnCall has an advanced Slack App with dozens of features making it even possible for users to be on-call and work with 
+alerts completely inside Slack. The drawback is that our Slack bot requires a lot of permissions and
+some of those permissions may sound suspicious, so we commented on them to give you more context.
+#### Content and info about you
+The bot is using those permissions to receive Slack handles and avatars. 
+Those permissions are supporting account matching between Grafana and Slack. 
+- **View information about your identity**
+- **View profile details about people in your workspace**
+#### Content and info about channels & conversations
+- **View basic information about public channels in your workspace** 
+— this permission is supporting channel selectors in the integration settings so the user could choose where to 
+send Alert Groups.
+- **View messages and other content in public channels, private channels, direct messages, and group direct messages 
+that Grafana OnCall has been added to** — this permission is supporting a feature of adding messages to the resolution 
+notes in the Alert Group's Slack thread.
+- **View basic information about private channels that Grafana OnCall has been added to** — this permission allows to 
+add a slack bot to the private channel and make it selectable in the list of channels. 
+So users will be able to route Alert Groups to the private channels.
+- **View basic information about direct messages that Grafana OnCall has been added to**
+#### Content and info about your workspace
+This set of permissions is supporting the ability of Grafana OnCall to match users with Grafana users.
+- **View people in your workspace**
+- **View email addresses of people in your workspace**
+- **View the name, email domain, and icon for workspaces Grafana OnCall is connected to**
+- **View user groups in your workspace**
+- **View profile details about people in your workspace**
+#### Perform actions as you
+- **Send messages on your behalf** — this permission may sound suspicious, but it's actually a general ability 
```

Comment:
May want to reiterate here: Grafana OnCall will not impersonate or post using your handle to slack it will always post as the bot.

Reply:
"Send messages on your behalf" is smth Slack adds, we can't change this text.